etc/rose.conf.example

```diff
@@ -19,6 +19,9 @@ meta-path=DIR1[:DIR2[:...]]
 #
 # URL to Rose documentation.
 rose-doc=http://metomi.github.io/rose/
+#
```

Comment:
This will create a leading blank line in the docs.

Reply:
Should now be fixed.